### ISSUE TITLE
discussion, Kernel

### ISSUE THREAD

**Issue by aseungbo** [aseungbo/TMDB-Box-office-Prediction#12]

**Comment by dada9140** [comment]
1. 달러표시가 조정된 인플레이션 또는 명목인가? 
(Are they inflation adjusted or nominal? Because some of them seem to be, like the 1925 Metropolis movie, and others no)

2. 단위가 어떤 것은 millions로 표현 다른 것은 thousands로 표현되어 있는 것 같음 
(There are also some values that there are clearly expressed in millions and others that are expressed in thousands, is the test set affected by the same problem?)

->대부분 단위  USD 하지만 몇개가 million 으로표현되어있음

3. 훈련세트에 백개정도의 영화가 수입이 이상하게 표현되어 있음. 백만을 곱해야 하거나 천을 곱해야 함. 수입 고칠 때 또는 제거할 때 점수 올라감
(There are around a hundred movies in the training set that have weird revenues. Some have a revenue that needs to be multiplied with 1e6 and others with 1e3, verified using revenue from IMDB. But when I try to fix the revenue for these movies my score goes up. Same thing happens if I remove them. Have I misunderstood something about the revenue? Is there a correlation I not seeing?) 

4. 예산에서도 위와 같은 문제 나타남 
(For the film Zyzzyx road, its revenue is 30 and its unit is in million, so you have to multiply by 1000000 its revenue. The problem also appears for the budget variable. One of the big problems that will not be solved is in the test sample when the budget and income variables do not have the same unit (e. g. The Karate Kid, Part II 1986). It is up to us to minimize this problem ;)) 

5. “World wide" or "US Canada" revenue 
 "The Devil's Violinist",  Eight Below 등 많은 영화가 US  revenue 섞여있음  
If I fix all values in Train data  (update all values to World or US revenue) it will not help if test data contains same errors.

**Comment by dada9140** [comment]
<외부데이터에 관하여> 
-영화 개봉전 데이터소스 만 사용할 것  
movie ratings
로튼토마토 점수(Rotten Tomato score) : 영화의 개봉 좌우할 수 있음  
Ex) 비평가들로부터 호평받은 경우 일찍 개봉 
평점은 개봉전후로 극적으로 바뀌는 경향은 없다. (예를 들어, 로튼토마토 점수는 개봉 당일 rating에서 5% 이상 변동하는 경우는 거의 없음/잠재적으로 입소문 대용으로 사용될 수도 있음)
궁극적으로 그 해석은 영화가 개봉되기 전에 어떻게 데이터를 이용할 수 있는지 여부에 달림..

Popularity 해석에 관하여 개봉전 인기도인지 개봉후 인기도인지 자신의 해석하기 나름
